integration/ig/k8s/trace_exec_test.go

```diff
@@ -48,9 +48,10 @@ func TestTraceExec(t *testing.T) {
 						WithContainerImageName("docker.io/library/busybox:latest", isDockerRuntime),
 						WithPodLabels("test-pod", ns, isCrioRuntime),
 					),
-					Comm: "sh",
-					Args: shArgs,
-					Cwd:  "/",
+					Comm:  "sh",
+					Pcomm: "containerd-shim",
```

Comment:
@mauriciovasquezbernal Do you have an idea how to normalise this in a way that works for cri-o?